packages/to-json-schema/src/type.ts

```diff
@@ -0,0 +1,22 @@
+import type * as v from 'valibot';
+
+/**
+ * JSON Schema config type.
+ */
+export interface JsonSchemaConfig {
+  /**
+   * Whether to force conversion to JSON Schema even for incompatible schemas and actions.
+   */
+  force?: boolean;
+  /**
+   * The name of the definitions property used. Defaults to "$defs".
+   */
+  definitionPath?: '$defs' | 'definitions';
```

Comment:
´definitions´ is the old name but JSON schema draft07 supports both and a lot of tooling tend to use the old name.

I don't have concrete example so we can remove this config and wait for users to file issues on that !

Reply:
I am not familiar enough with JSON schema to know what is best. If you have a strong opinion on this, feel free to share it. Otherwise we can add it in the future if needed.